stack-easybib/spec/role-nginxapp_spec.rb

```diff
@@ -88,21 +88,6 @@
             include("access_log #{access_log};")
           )
       end
-
-      describe 'pools' do
-        before do
-          node.override['php-fpm']['pools'] = %w(www1 www2 www3)
-        end
-
-        it 'creates three upstreams' do
```

Comment:
@baccenfutter My response was regarding the "configuring the version" via node attribute.